cli/cli-client/src/main/java/org/infinispan/cli/commands/Dec.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.cli.commands;
+
+import org.aesh.command.Command;
+import org.aesh.command.CommandDefinition;
+import org.aesh.command.CommandResult;
+import org.aesh.command.impl.completer.BooleanOptionCompleter;
+import org.aesh.command.option.Argument;
+import org.aesh.command.option.Option;
+import org.infinispan.cli.activators.ConnectionActivator;
+import org.infinispan.cli.completers.CounterCompleter;
+import org.infinispan.cli.impl.ContextAwareCommandInvocation;
+import org.kohsuke.MetaInfServices;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.0
+ **/
+@MetaInfServices(Command.class)
+@CommandDefinition(name = Dec.CMD, description = "Decrements a counter", activator = ConnectionActivator.class, aliases = "decrement")
+public class Dec extends CliCommand {
```

Comment:
Do we want a count to decrement by as well? Or do we only allow decrementing by one?

Reply:
They mirror the actual counter ops. You use add for variable deltas